CMakeLists.txt

```diff
@@ -280,12 +280,24 @@ if (BUILD_CPU)
         include(CheckCXXCompilerFlag)
         check_cxx_compiler_flag(-mavx512f HAS_AVX512F_FLAG)
         check_cxx_compiler_flag(-mavx512bf16 HAS_AVX512BF16_FLAG)
+        check_cxx_compiler_flag(-mavx512dq HAS_AVX512DQ)
+        check_cxx_compiler_flag(-mavx512bw HAS_AVX512BW)
+        check_cxx_compiler_flag(-mavx512vl HAS_AVX512VL)
```

Comment:
I don't think we need to bother testing for these and can just safely add them when adding `-mavx512f`.